src/kw_db.sh

```diff
@@ -51,3 +51,30 @@ function execute_command_db()
 
   sqlite3 "$db" -batch "$sql_cmd"
 }
+
+# This function inserts values into table of given database
+#
+# @table:     Table to insert data into
+# @entries:   Columns on the table where to add the data
+# @values:    Rows of data to be added
+# @db:        Name of the database file
+# @db_folder: Path to the folder that contains @db
+#
+# Return:
+# 2 if db doesn't exist;
+# 0 if succesful; non-zero otherwise
+function insert_into()
+{
+  local table="$1"
+  local entries="$2"
+  local values="$3"
+  local db="${4:-"$DB_NAME"}"
+  local db_folder="${5:-"$KW_DATA_DIR"}"
+  local db_path
+
```

Comment:
Because of the way SQL commands work `values` cannot be empty. The way you add an empty value is by either not specifying the specific column, or adding a literal (`''`) empty value. With this in mind I'll validate `table` and `values`. The `entries` variable can be empty if you are adding values to all columns.